ilpycon/templates/site_base.html

```diff
@@ -16,45 +16,48 @@
     {% block extra_head %}{% endblock %}
 {% endblock %}
 
-{% block body_extra_attributes %}{% if LANGUAGE_BIDI %}dir="rtl"{% endif %}{% endblock %}
+{% block body_extra_attributes %}{% endblock %}
 
 {% block topbar_base %}
-    
-<header {% block extra_header_class %}class="headerInnerPage"{% endblock %}>
-    <div class="navbar {% block navbar_class %}navbar-expand-lg navbar-light{% endblock %}">
-        <div class="container">
-        {% block topbar %}
-            {% block site_brand %}<a class="navbar-brand" href="{% url "home" %}">{{ SITE_NAME }}</a>{% endblock %}
-            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent">
-                <span class="fa fa-bars"></span>
-            </button>
-
-            <div class="collapse navbar-collapse" id="navbarSupportedContent">
-                {% block nav %}
-                    <ul class="navbar-nav {% if LANGUAGE_BIDI %}ml-auto{% else %}mr-auto{% endif %}">
-                        {% block nav_items %}
-                        {% sitetree_menu from "main" include "trunk" %}
-                        {% if request.user.is_authenticated %}
-                            <li class="navbar-item">
-                                <a class="nav-link" href="{% url "dashboard" %}"><i class="fas fa-tasks"></i> {% trans "Dashboard" %}</a>
-                            </li>
-                        {% endif %}
-                        {% endblock %}
-                    </ul>
-                {% endblock %}
-                {% block account_bar %}{% include "_account_bar.html" %}{% endblock %}
-                {% block language_bar %}{% include "_language_bar.html" %}{% endblock %}
-            </div>
-        {% endblock %}
-        </div>
+
+<div {% block extra_header_class %}class="headerInnerPage"{% endblock %}>
+  {% block headerInnerPages %}<div id="logoInnerPage"></div>{% endblock %}
+    <div class="navPyconIL" {% if LANGUAGE_BIDI %}dir="rtl"{% endif %}>
```

Comment:
We started out with `dir="rtl"` on the `body` -- @eldan, can you explain why you changed it?

Reply:
The `if LANGUAGE` thing was originally in `body_extra_attributes`. However if I move it back to there, then RTL seems to destroy the design of the topbar (e.g. boat and PyConIL icon get misaligned). I believe that's the reason Eldan removed it.
I think the solution for this case should be to have the graphical parts that require LTR force that explicitly.

There are several issues to decide here:
1. Should we put directionality at root (html) level? I say YES. This would fix the title issue. Note that currently, when you are in the hebrew homepage, the title (title tag in the header block. the one which which browsers display on their titlebar) has bad directionality.
2. Should we put it in html or css? If Meir says css I'd follow his advice.
3. Assuming we do move directionality to <body> or higher level: should look at the topbar code and see if we have to force LTR or use other means.